types/rlp_unmarshal.go

```diff
@@ -34,6 +34,26 @@ func UnmarshalRlp(obj unmarshalRLPFunc, input []byte) error {
 	return nil
 }
 
+func (t *TxType) UnmarshalRLPFrom(p *fastrlp.Parser, v *fastrlp.Value) error {
```

Comment:
Does it make sense to cover this function with a unit test?

Reply:
We already have tests for the high-level function. Don't really see a reason why we should duplicate that test.

Reply:
For example, `TestRLPMarshall_And_Unmarshall_TypedTransaction` calls `UnmarshalRLP` that uses `unmarshalRLPFrom` inside.

Reply:
Nope, it extracts `TxType` using `txTypeFromByte`
(https://github.com/0xPolygon/polygon-edge/pull/1114/files#diff-efc5999399a6ff1082b29a52ce57fd7bfd6ebe4789d686a44c70523d2a61c684R253) and it sets offset so that `txType` doesn't get unmarshaled in `unmarshalRLPFrom` function...